src/lib/rsa/sandboxed/RSAKeysIframe.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>RSA Keys Iframe | Keyguard Demo</title>
```

Comment:
It's not only used in the demos.
```suggestion
    <title>RSA Keys Iframe</title>
```

Reply:
Resolved in #524